operators/src/source/ogr_source/mod.rs

```diff
@@ -610,6 +610,64 @@ impl FeaturesProvider<'_> {
         Ok(())
     }
 
+    ///  Creates a time filter string.
+    fn create_time_filter_string(
```

Comment:
can you add tests for the different cases that can occur?

Reply:
https://github.com/geo-engine/geoengine/pull/961/commits/50d34cbb5e643cc87cb05bd08ba843180e81f081